websites/A/AnimeVost/presence.ts

```diff
@@ -0,0 +1,181 @@
+const presence = new Presence({
+	clientId: "1096792677888053308",
+});
+
+async function getStrings() {
+	return presence.getStrings({
+		play: "general.watchingVid",
+		pause: "general.paused",
+	});
+}
+
+let video = {
+		duration: 0,
+		currentTime: 0,
+		paused: true,
+	},
+	strings: Awaited<ReturnType<typeof getStrings>>;
+
+const animetypes: { [key: string]: string } = {
+		tv: "ТВ-сериал",
+		"tv-speshl": "ТВ-спешл",
+		ova: "OVA",
+		ona: "ONA",
+		"polnometrazhnyy-film": "Фильм",
+		"korotkometrazhnyy-film": "Фильм",
+		dunkhua: "Дунху",
+	},
+	animegenres: { [key: string]: string } = {
+		"boyevyye-iskusstva": "Боевые искусства",
+		voyna: "Война",
+		drama: "Драма",
+		detektiv: "Детектив",
+		istoriya: "История",
+		komediya: "Комедия",
+		mekha: "Меха",
+		mistika: "Мистика",
+		"makho-sedze": "Махо-сёдзё",
+		muzykalnyy: "Музыкальный",
+		povsednevnost: "Повседневность",
+		priklyucheniya: "Приключения",
+		parodiya: "Пародия",
+		romantika: "Романтика",
+		senen: "Сёнен",
+		sedze: "Сёдзё",
+		sport: "Спорт",
+		skazka: "Сказка",
+		"sedze-ay": "Сёдзё-ай",
+		"senen-ay": "Сёнен-ай",
+		samurai: "Самураи",
+		triller: "Триллер",
+		uzhasy: "Ужасы",
+		fantastika: "Фантастика",
+		fentezi: "Фентези",
+		shkola: "Школа",
+		etti: "Эччи",
+	};
+let strtstamp = Math.floor(Date.now() / 1000),
+	pausestamp = false;
+
+presence.on(
+	"iFrameData",
+	(data: { duration: number; currentTime: number; paused: boolean }) => {
+		video = data;
+	}
+);
+
+presence.on("UpdateData", async () => {
+	const [privacy, time, logo, buttons] = await Promise.all([
+			presence.getSetting<boolean>("privacy"),
+			presence.getSetting<boolean>("time"),
+			presence.getSetting<boolean>("logo"),
+			presence.getSetting<boolean>("buttons"),
+		]),
+		presenceData: PresenceData = {
+			details: "Где-то на сайте",
+			largeImageKey: "https://i.imgur.com/9KbVqtj.png",
+			smallImageText: "🏴‍☠️ AnimeVost",
+		},
+		websiteloc = document.location.pathname.split("/");
+	if (!strings) strings = await getStrings();
+	if (!privacy) {
+		if (websiteloc[1] === "") presenceData.details = "На главной странице";
+		if (websiteloc[1] === "zhanr") {
+			const animegenre = animegenres[websiteloc[2]];
+			presenceData.details = `🔎 В поисках аниме жанра ${animegenre}`;
+			presenceData.smallImageKey = websiteloc[2];
```

Comment:
`smallImageKey` should use Imgur links rather than Discord asset names. This Presence uses many assets, and should put them in an enum:
https://github.com/PreMiD/Presences/blob/3aa7e3fb8e54999012de1d1528f37bc907ae4213/websites/C/Crunchyroll/presence.ts#L52-L58

Reply:
Not resolved.

Reply:
Sry, my bad, changed it.